modules/admin-ui-picard/app/src/thunks/seriesDetailsThunks.js

```diff
@@ -0,0 +1,250 @@
+                    },
+                    {
+                        type: 'rss',
+                        version: '2.0',
+                        link: typeLink[0] + 'rss' + versionLink[0] + '2.0' + versionLink[1]
+                    }
+                ]
+            }
+        }
+
+        dispatch(loadSeriesDetailsFeedsSuccess(seriesFeeds));
+
+    } catch (e) {
+        dispatch(loadSeriesDetailsFailure());
+    }
+}
+
+// fetch theme of certain series from server
+export const fetchSeriesDetailsTheme = id => async dispatch => {
+    try {
+        dispatch(loadSeriesDetailsInProgress());
+
+        let data = await axios.get(`admin-ng/series/${id}/theme.json`);
+
+        const themeResponse = await data.data;
+
+        let seriesTheme = '';
+
+        // check if series has a theme
+        if (!_.isEmpty(themeResponse)) {
+            // transform response for further use
+            seriesTheme = (transformToIdValueArray(themeResponse))[0].value;
+        }
+
+        dispatch(loadSeriesDetailsThemeSuccess(seriesTheme));
+    } catch (e) {
+        dispatch(loadSeriesDetailsFailure());
+    }
+}
+
+// fetch names of possible themes from server
+export const fetchNamesOfPossibleThemes = () => async dispatch => {
+    try {
+        dispatch(loadSeriesDetailsThemeNamesInProgress());
+
+        let data = await axios.get('/admin-ng/resources/THEMES.NAME.json');
+
+        const response = await data.data;
+
+        // transform response for further use
+        let themeNames = transformToIdValueArray(response);
+
+        dispatch(loadSeriesDetailsThemeNamesSuccess(themeNames));
+    } catch (e) {
+        dispatch(loadSeriesDetailsThemeNamesFailure());
+    }
+}
+
+// update series with new metadata
+export const updateSeriesMetadata = (id, values) => async (dispatch, getState) => {
+    try {
+        let metadataInfos = getSeriesDetailsMetadata(getState());
+
+        let fields = [];
+        let updatedFields = [];
+
+        metadataInfos.fields.forEach(field => {
+            if (field.value !== values[field.id]) {
+                let updatedField = {
+                    ...field,
+                    value: values[field.id]
+                }
+                updatedFields.push(updatedField);
+                fields.push(updatedField);
+            } else {
+                fields.push({...field});
+            }
+        });
+
+        let data = new URLSearchParams();
+        data.append("metadata",JSON.stringify([{
+            flavor: metadataInfos.flavor,
+            title: metadataInfos.title,
+            fields: updatedFields
+        }]));
+
+        await axios.put(`/admin-ng/series/${id}/metadata`, data);
+
+        // updated metadata in series details redux store
+        let seriesMetadata = {
+            flavor: metadataInfos.flavor,
+            title: metadataInfos.title,
+            fields: fields
+        };
+        dispatch(loadSeriesDetailsMetadataSuccess(seriesMetadata));
```

Comment:
See also comment in modules/admin-ui-picard/app/src/reducers/seriesDetailsReducers.js
Either add `dispatch(loadSeriesDetailsInProgress())` and `dispatch(loadSeriesDetailsFailure())` here, or don't change `isLoading` for this action in the reducer